app/controllers/concerns/find_common.rb

```diff
@@ -11,7 +11,7 @@ def find_resource
   def resource_finder(scope, id)
     raise ActiveRecord::RecordNotFound if scope.empty?
     result = scope.from_param(id) if scope.respond_to?(:from_param)
-    result ||= scope.friendly.find(id) if scope.respond_to?(:friendly)
+    result ||= scope.friendly.find(id) rescue nil if scope.respond_to?(:friendly)
```

Comment:
This raises ResourceNotFound if the friendly find doesn't find anything, causing the `scope.find(id)` attempt to never run for models with friendly id.